source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,55 @@
+=================
+Insert a Document
+=================
+
+.. default-domain:: mongodb
+
+You can insert a single document into a collection using the ``insertOne()``
+method on a ``MongoCollection`` object. To insert a document, construct a
+``Document`` object that contains the fields and values that you want to
+store. If you call the ``insertOne()`` method on a collection that does
+not exist yet, the method creates it for you.
+
+For more information on inserting data using MongoDB with the Java driver,
+see our guide (TODO: Fundamentals > CRUD Operations > Write Operations)
+on inserting documents.
+
+Upon successful insert, ``insertOne()`` returns an instance of
+``InsertOneResult`` which contains information such as the id (also known
+as the ``ObjectId`` that you can retrieve with the ``getInsertedId()``
+method. If you specified a value for the ``_id`` field in the inserted
+document, ``getInsertedId()`` returns null.
```

Comment:
Did you test this assertion?  When I try it, it returns the value of the _id from the document.

Reply:
Strange, I don't recall where I had read that, but should have tested. I'll remove it now.